includes/CLI/Plugin_Check_Command.php

```diff
@@ -271,21 +404,37 @@ private function get_options( $assoc_args ) {
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @param array $assoc_args Associative arguments.
+	 * @param array $assoc_args     Associative arguments.
+	 * @param array $default_fields Default fields.
 	 * @return WP_CLI\Formatter The formatter instance.
 	 */
-	private function get_formatter( $assoc_args ) {
+	private function get_formatter( $assoc_args, $default_fields ) {
+		if ( isset( $assoc_args['fields'] ) ) {
+			$default_fields = wp_parse_args( $assoc_args['fields'], $default_fields );
+		}
+
+		return new WP_CLI\Formatter(
+			$assoc_args,
+			$default_fields
+		);
+	}
+
+	/**
+	 * Returns check default fields.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $assoc_args Associative arguments.
+	 * @return array Default fields.
+	 */
+	private function get_check_default_fields( $assoc_args ) {
```

Comment:
@felixarntz I have created this separate private method to reduce `Cyclomatic complexity` and to make `check()` method leaner.